packages/radio-button-react/src/RadioButton.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { useState, forwardRef, InputHTMLAttributes, ChangeEventHandler } from "react";
+import React, { useState, forwardRef, InputHTMLAttributes, ChangeEventHandler, ReactNode } from "react";
 import cx from "classnames";
 import { nanoid } from "nanoid";
 import { useRadioGroupContext } from "./radioGroupContext";
 
 export interface RadioButtonProps extends Omit<InputHTMLAttributes<HTMLInputElement>, "children"> {
+    children?: ReactNode;
     label: string;
```

Comment:
Kan gjerne gjøre denne optional i samme slengen, så slipper man tom string som prop :)

Reply:
Får du en situasjon der du kan droppe både label og children og få en ugyldig radio-button (uu-messig vertfall).

Reply:
Ja, var litt usikker på den der. Dumt om begge er optional og så dropper du begge

Reply:
Vet at et delmål var å få et likere API sammenlignet med `Checkbox`, men kanskje vi da skal utsette det å legge til `children` som prop? Så kan heller typen bli `label: ReactNode` for å løse det umiddelbare behovet deres.